src/openzaak/config/tests/bootstrap/test_load_authorizations.py

```diff
@@ -0,0 +1,326 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2024 Dimpact
+
+import tempfile
+from pathlib import Path
+
+from django.core.management import call_command
+from django.test import TestCase, override_settings
+
+import yaml
+from vng_api_common.authorizations.models import Applicatie, Autorisatie
+from vng_api_common.constants import ComponentTypes, VertrouwelijkheidsAanduiding
+from vng_api_common.models import JWTSecret
+
+from openzaak.components.autorisaties.models import AutorisatieSpec
+from openzaak.components.autorisaties.tests.factories import (
+    ApplicatieFactory,
+    AutorisatieFactory,
+    AutorisatieSpecFactory,
+)
+from openzaak.components.besluiten.api.scopes import SCOPE_BESLUITEN_ALLES_LEZEN
+from openzaak.components.zaken.api.scopes import (
+    SCOPE_ZAKEN_ALLES_LEZEN,
+    SCOPE_ZAKEN_CREATE,
+)
+from openzaak.config.bootstrap.authorizations import AuthorizationConfigurationStep
+
+ZAAKTYPE1 = "https://acc.openzaak.nl/zaaktypen/1"
+ZAAKTYPE2 = "https://external.acc.openzaak.nl/zaaktypen/2"
+ZAAKTYPE3 = "https://acc.openzaak.nl/zaaktypen/3"
+ZAAKTYPE4 = "https://acc.openzaak.nl/zaaktypen/4"
+
+
+class AuthorizationConfigurationTests(TestCase):
+    maxDiff = None
+
+    def setUp(self):
+        super().setUp()
+
+        self.temp_dir = tempfile.TemporaryDirectory()
+        self.dump_path = str(Path(self.temp_dir.name) / "auth_dump.yaml")
```

Comment:
I think it could be nice to have a real `auth_dump.yaml` file added to the test folder, so we could see its content as an example. The same applies for `mapping.yaml` file

Reply:
I removed the tempfiles and added two static .yaml files